src/main/views/subscription-config-list-language.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "template.njk" %}
+{% from "macros/common-components.njk" import goBack, submitButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ super() }}
+    {{ goBack(
+        path= '/subscription-configure-list',
+        text = backButton,
+        clickEvent = "history.back();")
+    }}
+{% endblock %}
+
+{% block content %}
+    <div class="parent-box">
+        {% if noSelectionError %}
+            {{ govukErrorSummary({
+                titleText: error.title,
+                errorList: [
+                    {
+                        text: error.noSelection,
+                        href: "#english"
+                    }
+                ]
+            }) }}
+        {% endif %}
+        <form action="subscription-configure-list-confirmed" method="post">
```

Comment:
We nornally post back to the same page to check for error and not the confirmed page. Might be better to add a post method to subscription-config-list-language page for that

Reply:
fixed.